lib/interfaces/ITrendingMoviesBloc.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:tmdb_movies/models/item_model.dart';
+
+import 'bloc.dart';
+
+abstract class ITrendingMovies extends Bloc{
+  Stream<ItemModel> get stream;
+  void fetchTrendingMovies();
+}
```

Comment:
Remember to use ctrl+alt+l, you missed a newline at end of file here.

Reply:
Done!